data/resources/style.css

```diff
@@ -15,15 +15,23 @@
 }
 
 .chat-list row .unread-count {
-  background-color: @accent_bg_color;
-  color: @accent_fg_color;
   font-size: 0.8em;
   font-weight: bold;
   min-width: 0.7em;
   border-radius: 10px;
   padding: 2px 5px;
 }
 
+.chat-list row .unread-count-unmuted {
+  background-color: @accent_bg_color;
+  color: @accent_fg_color;
+}
+
+.chat-list row .unread-count-muted {
+  background-color: @dark_1;
```

Comment:
This color is ok for dark theme, but I find it too dark for the light theme. I find `@light_5` to be better for light.

Reply:
If I understand you correctly, we want to make  a case distinction and use `@light_5` for the light variant and `@dark_1` for the dark variant. `@light_5` would look too bright IMO.

![light_5](https://user-images.githubusercontent.com/3630213/140622475-d2f21b45-6982-47bc-b34a-bf45eb777d17.png)
![dark_1](https://user-images.githubusercontent.com/3630213/140622474-7dce1acf-9408-4bfd-8962-ccd5a236c546.png)

I'll revert to just `@light_5`, if you don't agree.

Reply:
> we want to make a case distinction

Yep, it's fine now.